packages/vda-node-client/src/getDataCenters.ts

```diff
@@ -0,0 +1,98 @@
+import { BigNumberish } from "ethers";
+import { executeFunction } from "./utils";
+import { EnumStatus } from "@verida/types";
+
+/**
+ * Call getDatacenters() function of `StorageNodeRegistry` contract
+ * @param ids Array of datacenterIds
+ */
+export async function isRegisteredDataCenterName(network: string, name: string) {
+    return await executeFunction(
+        network,
+        'isRegisteredDataCenterName',
+        'Failed to check datacenter name',
+        name
+    );
+}
+  
+/**
+ * Call getDatacenters() function of `StorageNodeRegistry` contract
+ * @param ids Array of datacenterIds
+ */
+export async function getDataCenters(network: string, ids: BigNumberish[]) {
+    return await executeFunction(
+        network,
+        'getDataCenters',
+        'Failed to get datacenters by id',
+        ids
+    );
+}
+
+/**
+ * Call getDatacentersByName() function of `StorageNodeRegistry` contract
+ * @param names Array of name
+ */
+export async function getDataCentersByName(network: string, names: string[]) {
```

Comment:
Update spelling of `Center` to be `Centre` for all the code

Reply:
Updated